### ISSUE TITLE
Save as CSV in History Data only exports ~2000 entries

### ISSUE THREAD

**Issue by gitisgreat2023** [ioBroker/ioBroker.history#309]
When selecting a custom range in History Data and "save data as csv" only a fraction of the selected time range is exported.

There's probably a limit on the amount of data displayed in the GUI, that's fine. But when I export data I would like to get the full data, because this export circumvents displaying many items.

How can I export more than 2000 entries, without increasing some default for displaying data points in history data?

**Comment by gitisgreat2023** [comment]
I looked at the raw data in the, but there's a file for every data. That would be cumbersome to merge many files into one single file. That IMHO would have been exactly the task of the export function.

**Comment by chris299** [comment]
same issue here

**Comment by chris299** [comment]
I just learned that it is not a bug, but a "feature", so works as designed....
see here:
https://forum.iobroker.net/topic/23730/history-adapter-csv-download-problem/23

**Comment by gitisgreat2023** [comment]
Yes it is indeed. I wrote a Node Red script to save the data without limitations to a csv file.
